docs/website/docs/overview/dev_and_deploy.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Develop and Deploy
+sidebar_position: 5
+---
+
+# Develop and Deploy
+
+The two most important commands in `odo` are `odo dev` and `odo deploy`. 
+
+In some situations, you'd want to use [`odo dev`](/docs/command-reference/dev) over [`odo deploy`](/docs/command-reference/deploy) and vice-versa. This document highlights when you should use either command.
+
+## When should I use `odo dev`?
+
+`odo dev` should be used in the initial development process of your application. 
+
+For example, you should use `odo dev` when you are:
+* Making changes constantly
+* Want to preview any changes
+* Testing initial Kubernetes support for your application
+* Want to debug and run tests
+* Run the application on a local development environment
```

Comment:
A small comment about the tense of the verbs, which I found hard to read in conjunction with "... when you are ".

I would suggest something like:
```
For example, you should use `odo dev` when:
* you are making changes constantly
* you want to preview any changes
* you are testing initial Kubernetes support for your application
* you want to debug and run tests
* you want to run the application on a local development environment
```

What do you think?

Reply:
IMO want to avoid repeating the same sentence twice starting with the same word

Reply:
I agree with Charlie on this one. It's much nicer to read with the current change, and it also makes more sense.

Reply:
Alright - maybe it's just me then :D

Reply:
I found the previous change hard to read at the time I reviewed this PR a few days back:

```
For example, you should use odo dev when you are:
- making changes constantly
- want to preview any changes
- ...
```

But it looks better now with the latest changes you suggested: 
```
For example, you should use odo dev when you are working with a local development environment and:
- making changes constantly
- want to preview any changes
- ...
```